### PR TITLE
v1.6.3

### PR DESCRIPTION
**Pull request by k-gerner**
Add CloudChoice functionality

### CODE REVIEW

**Review by Fondryext — COMMENTED** [review]
Sorry for not noticing this earlier, but I realized we need to up the version to 1.6.3 in the `package.json` files and re-run `npm i` to update `package-lock` as well. I think you can do that directly in the hotfix branch instead of making another dev branch.
Also, this can wait until you go to merge, but fyi that for pulling into main I'd pref a commit message formatted similarly to:
https://github.com/yext/search-ui-react/pull/457

**Review by vijay267 — APPROVED** [review]

**Pull request merged**
